### ISSUE TITLE
Respond to number indicator click as well as to x axis

### ISSUE THREAD

**Issue by dachinat** [IonDen/ion.rangeSlider#561]
I think that it would be great if slider would respond to clicks on numbers when using `grid: true`.

Right now if you click on an axis itself then point would be changed, however if you click on a grid number than it does not.

<img width="318" alt="screen shot 2018-06-26 at 1 23 37 am" src="https://user-images.githubusercontent.com/22872749/41876775-fddad3d4-78df-11e8-8991-62c099602faa.png">

_Another option would be to to increase y area of click point so that it would include numbers at the bottom._

**Comment by IonDen** [comment]
Hi, basically, this behavior depends on skin.
For example, you can add hidden `:after` element to line with big box area and it will read all clicks.

**Comment by dachinat** [comment]
I don't know about `::after` pseudo-element, but I have solved this differently. I was using the Round skin.

I just changed `.irs-line` to 

```
.irs-line {
    top: 36px;
    height: 35px;
    border-top: 4px solid #DEE4EC;
    cursor: pointer;
}
```

I have removed background and applied border, increased height from 4px to higher number.

If you ever find a time to give a better, more optimal solution to this, perhaps with pseudo-elements, I'd be very thankful.

One important note, I also added `pointer-events: none;` to `.irs-grid`.
